webpack.mix.js

```diff
@@ -52,5 +52,6 @@ mix.scripts([
     'js/solstice.cookies.js',
     'js/solstice.eventsmodal.js',
     'js/solstice.js',
-    'js/solstice.donate.js'
+    'js/solstice.donate.js',
+    'js/eclipsefdn.adopters.js'
 ], 'docs/dist/js/solstice.js');
```

Comment:
I'm not sure for this should I make it separate? 

In hugo theme I need to add it  separately since the shortcode is used before footer's <script> https://github.com/EclipseFdn/hugo-solstice-theme/pull/190/files#diff-974472aeeb3c0d4802bb4087cf02e9cf5591004b99361954801b10b64d7a1f0b

Reply:
This is fine! This allows us to confirm at a minimum that the code does indeed compile.